src/RepRap.cpp

```diff
@@ -314,8 +314,10 @@ constexpr ObjectModelTableEntry RepRap::objectModelTable[] =
 #endif
 #if HAS_MASS_STORAGE
 	{ "logFile",				OBJECT_MODEL_FUNC(self->platform->GetLogFileName()),					ObjectModelEntryFlags::none },
+	{ "logLevel",				OBJECT_MODEL_FUNC(self->platform->GetLogLevel()),						ObjectModelEntryFlags::none },
 #else
 	{ "logFile",				OBJECT_MODEL_FUNC_NOSELF(nullptr),										ObjectModelEntryFlags::none },
+	{ "logLevel",				OBJECT_MODEL_FUNC_NOSELF(nullptr),										ObjectModelEntryFlags::none },
```

Comment:
This assumes that in SBC mode, RRF never needs to know the logging level. I'm not sure that is correct, because AFAIR DCS uses RRF to evaluate expressions. In which case, RRF needs to know the logging level even though it doesn.t do the logging itself.